phpstan.neon

```diff
@@ -7,6 +7,3 @@ parameters:
   paths:
     - src
     - tests
-
-  # Allow instanceof checks, particularly in tests
-  checkAlwaysTrueCheckTypeFunctionCall: false
```

Comment:
I agree this check should not be disabled per se, and if possible the setting could be applied on a per directory basis: i.e. for `tests` which can by static analysis be proven to pass.  Though I'd quite like the double confirmation.